.env

```diff
@@ -0,0 +1,2 @@
+# The port for the development server
+PORT=8080
```

Comment:
should be added to .gitignore file

Reply:
Strange. I thought I've seen a working example of a checked in `.env` file for default environment variable values that can be overridden by `.env.local` and the likes. This does indeed not seem to work as I imagined, though. Changing it, thanks!

Reply:
Changed it and added an `.env.example` instead.

Reply:
The more you know. I also assumed this would do (and acted upon this assumption in the past) being used to e.g.
https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
https://symfony.com/doc/current/configuration.html#overriding-environment-values-via-env-local

Reply:
Yeah, interestingly the webpack dev server happily picked up the variables inside the `.env.local` inside the container, but obviously that didn't help with mapping them to the hosts ports.